ee/tabby-db/migrations/0024_github-provided-repos.up.sql

```diff
@@ -6,5 +6,10 @@ CREATE TABLE github_provided_repositories(
     name TEXT NOT NULL,
     git_url TEXT NOT NULL,
     active BOOLEAN NOT NULL DEFAULT FALSE,
-    FOREIGN KEY (github_repository_provider_id) REFERENCES github_repository_provider(id) ON DELETE CASCADE
+    created_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+    updated_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+    FOREIGN KEY (github_repository_provider_id) REFERENCES github_repository_provider(id) ON DELETE CASCADE,
+    CONSTRAINT `idx_vendor_id` UNIQUE (vendor_id)
```

Comment:
add constraint on `vendor_id x github_repository_provider_id` for ease of implementing upsert